README.md

```diff
@@ -19,7 +19,7 @@ The module can deploy an Intel Optimized AWS Databricks Workspace.
 
 <p align="center">
   <a href="https://www.databricks.com/blog/2022/09/13/faster-insights-databricks-photon-using-aws-i4i-instances-latest-intel-ice-lake">
-  <img src="https://github.com/intel/terraform-intel-aws-databricks-workspace/blob/main/images/aws-dbx-1.png" alt="Link" width="600"/>
+  <img src="https://github.com/intel/terraform-intel-aws-databricks-workspace/blob/main/images/aws-dbx-1.png?raw=true" alt="Link" width="600"/>
```

Comment:
Do you also need to change the image name to dbx-runtime.png?

Reply:
dbx-runtime.png is for the last perform data out of 3. Check line 40 for this file